server/server.js

```diff
@@ -114,6 +115,7 @@ async function initKubeClient() {
     kubeclient.addCustomResourceDefinition(pushApplicationCRD);
     kubeclient.addCustomResourceDefinition(androidVariantCRD);
     kubeclient.addCustomResourceDefinition(iosVariantCRD);
+    kubeclient.addCustomResourceDefinition(mobildeSecuritySerivce);
```

Comment:
typo in variable name:

```suggestion
kubeclient.addCustomResourceDefinition(mobileSecurityService);
```